### PR TITLE
Work in progress on microreact server.

### PR DESCRIPTION
**Pull request by JSvoldgaard**
The openidconnect part in the json-file hasn't been tested to work yet.
Remember to add dev2.sofi-platform.dk in hosts-file.
Microreact Image can be found using guide sent by Mischa 27/5.

### CODE REVIEW

**Comment by sjkp** [comment]
Got it working. 
Things to notice:
docker-compse
`
Must override: ./microreact_defaults.json:/app/config.json (the docker image contains a existing config.json 
NEXTAUTH_URL_INTERNAL: http://localhost:3000/ <-- this must be the localhost name or you end in a redirect loop
`
microreact_defaults.json
`
"clientSecret": "can-be-any-thing-not-used", <-- this is not used, but it can't be empty
"checks":[] <-- if this is set you get authentication fails, it should be possible to configure this somehow
mailhog must have a username and password even though it doesn't check it
`

File: docker-compose.local.yml [hunk 1]
```diff
@@ -97,3 +97,20 @@ services:
...
       - "8025:8025"
+
+  microreact:
+    build:
+      context: ./microreact
+      dockerfile: Dockerfile
+    environment:
+      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/ca-certificates.crt
+      NEXTAUTH_URL: https://dev2.sofi-platform.dk/
+      NEXTAUTH_URL_INTERNAL: http://localhost:3000/
```
Comment (allanhvam) [inline docker-compose.local.yml:108]: I had to change this to `http://microreact:3000/` to get it working.

**Comment by allanhvam** [comment]
Session name does not seem to resolve (is `null`):
image

**Comment by allanhvam** [comment]
At some point I think we need to make `mapboxApiAccessToken` and the next auth secret configurable through environment variables.

**Pull request merged**